char/liberate/Makefile

```diff
@@ -0,0 +1,96 @@
+VPATH:=make
+LOG:=./LOG
+RUN_DIR:=$(shell pwd)
+SHELL:=bash
+
+.DEFAULT: all
+.PHONY: all do run clean purge
+
+CORNER:=TT
+TEMP:=25
+VLTG:=3.3
+ARGS:=-ccs
+VARIANT:=12T
+
+USER:=$(shell whoami)
+USER_MAIL:=$(shell echo `ldapsearch -x -LL "(uid=$(USER))" mail | sed -n "s/^mail: //p"`)
+export
+
+define \n
+
+
+endef
+
+ALL_CNR = TT \
+		FF \
+		SS \
+		FS \
+		SF
+
+setup:
+	echo "Setting things up...."
+	@mkdir -p $(VPATH) $(LOG) NETLIST LIBRARY LDB DATASHEET VERILOG
+	@cp ../techfiles/*.hspice MODELS/
+	@cp ../../lib/$(VARIANT)/spice/* NETLIST/
+	#@cd MODELS && sh ../sedSpice
+	@ls NETLIST/*spice |  sed 's/.spice\|NETLIST\///g' > celllist
+	@sed -i '/fill/d' celllist
+	@touch $(VPATH)/$@
+
+all:
+	echo "Entering all...."
+	@for i in $(ALL_CNR); do \
+		echo "Generating corner $$i"; \
+		$(MAKE) -f $(lastword $(MAKEFILE_LIST)) do CORNER=$$i; \
+	done
+
+run: setup
+	export CORNER=$(CORNER); \
+	export TEMP=$(TEMP); \
+	export VLTG=$(VLTG); \
+	export ARGS=$(ARGS); \
+	export VARIANT=$(VARIANT); \
+	export NAME=`SCRIPTS/getname.tcl`; \
+	printf "*\n.LIB 'sm141064.hspice' nmos_3p3_t\n.LIB 'sm141064.hspice' pmos_3p3_t" > MODELS/include.sp; \
+	liberate --trio char.tcl |& tee LOG/$$NAME.log; \
+	lc_shell -no_log -f SCRIPTS/lib_to_db.tcl | tee out.txt
+	@cp *.lib* ../techfiles/
+	@cp *.db* ../techfiles/
+ifndef NO_MAIL
+	@echo "" | mail -s "Lib extraction finished for $(CORNER) $(TEMP) $(VLTG) $(ARGS) on `uname -n`" $(USER_MAIL)
+endif
+
+do: run
+	@cp *.lib* ../techfiles/
+	@cp *.db* ../techfiles/
+	@cp -r DATASHEET ../techfiles/
+	@cp -r VERILOG ../techfiles/
+	@mv *.lib LIBRARY/
+	@mv *.db LIBRARY/
+
+clean:
```

Comment:
are those in `.gitignore` already?

Reply:
not sure what you mean here..think its okay.  This is just moving final .lib file.